### DISCUSSION TITLE
Wallet with multiple keys

### DISCUSSION THREAD

**Discussion by j4r3kb** [Chia-Network/chia-blockchain#7779] [Q&A]
I have a wallet running not problems fully synced. Just added another set of keys with `chia keys add`.

Now the `chia wallet show` for the first key shows wallet synced, but the same for the new key is not synced and at height 0 and is not picking up blocks for several minutes.

Do we have to sync each key in a wallet separately? Is it even doable on the same instance or I have to run it on it's own instance as a default key **just to send a transaction over the network**?

**Comment by keliew** [comment]
Each key needs to be synced separately. Apparently, it'd take your local node and sync, but I can't seem to find the FAQ on Wiki.

The wallet which will send/receive transaction will always need to be synced first.

Once light wallets are in place, I think it would be better.
